Add a way to exclude specific output

vesim987 opened the pull request:
This adds ability to exclude specific output for a bar with `!` character, for example:
```
[
  {
    "layer": "bottom",
    "output": "eDP-1",
    # config for main display
  },
  {
    "layer": "bottom",
    "output": "!eDP-1",
    # config for extra displays
  }
]
```

Alexays commented on src/client.cpp:
```diff
@@ -65,21 +65,25 @@ void waybar::Client::handleOutput(std::unique_ptr<struct waybar_output> &output)
-  bool found = true;
+  auto check_output = [](std::string_view                       config_output_name,
+                         std::unique_ptr<struct waybar_output> &output) {
+    if (config_output_name.length() > 0) {
```
```suggestion
    if (!config_output_name.empty()) {
```

Alexays replied:
Can you avoid using another function and don't return when the result is false as it'll skip others screen.

vesim987 replied:
WDYM by another function? I've used lambda to reduce duplication of that code.

Alexays commented:
Otherise, LGTM, pretty simple with a !
Could you update the man and the wiki? 😃

vesim987 commented:
Rebased against latest master, and also added info to man and [wiki](https://github.com/Alexays/Waybar/wiki/Configuration#configuration-of-multiple-outputs).

Alexays commented on src/client.cpp:
```diff
@@ -64,21 +64,31 @@ void waybar::Client::handleOutput(struct waybar_output &output) {
-      if (output_conf.isString() && output_conf.asString() == output.name) {
-        in_array = true;
-        break;
+      if (output_conf.isString()) {
```
```suggestion
      if (output_conf.isString() && check_output(output_conf.asString(), output)) {
```

Alexays replied:
Mb we should not do this check here, but only when only one screen name is passed, otherwise when there is
["!edp1", "edp2"] all screens will enter the config

Alexays commented:
> Mb we should not do this check here, but only when only one screen name is passed, otherwise when there is
> ["!edp1", "edp2"] all screens will enter the config

Did you see this comment :?

vesim987 commented:
Ouch, somehow I've missed that comment, sorry.
Yep, that's good point.

Alexays commented on src/client.cpp:
```diff
@@ -64,21 +64,25 @@ void waybar::Client::handleOutput(struct waybar_output &output) {
   }
-  return found;
+
+  return false;
```
Should be true by default, when no output is defined in the config file.
